### DISCUSSION TITLE
want to use orchardcore in my microservices

### DISCUSSION THREAD

**Discussion by livehop** [OrchardCMS/OrchardCore#12370] [Q&A]
Hi, I am wanting to use orchardcore as a headless CMS sitting as one microservice accessible via a "preconfigured" endpoint for both admin and graphql queries; that way I can configure the urls cleanly in my API Gateway. 
However, when I am unable to find an easy way to map a preconfigured path; also when I run the project by adding the nuget packages I get an error when trying to get to the admin page -- saying multiple path matched.

**Comment by livehop** [comment]
Also ... I think the same code works with .Net core 5.0; version 1.0.0
If I created a new project with .Net 6; I think we see this issue

**Reply by Skrypt** [reply]
If you get route conflicts that's probably because the /admin path is already mapped inside your own custom code. You can change the default path of the admin by configuration. Else, it should definitely work.
